ocs_ci/deployment/deployment.py

```diff
@@ -247,3 +241,24 @@ def add_node(self):
         Implement platform specif add_node in child class
         """
         raise NotImplementedError("add node functionality node implemented")
+
+    def patch_default_sc_to_non_default(self):
+        """
+        Patch storage class which comes as default with installation to non-default
+        """
+        sc_to_patch = None
+        platform = config.ENV_DATA.get('platform')
+        if platform in constants.AWS_PLATFORM:
```

Comment:
I don't think we need to keep AWS and aws in some tuple as we can use just
if constants.AWS_PLATFORM == self.platform.lower()

We can think of changing self.platform in constructor to .lower() version directly than you can just compare to self.platform.